samples/core/ai-platform/ai-platform.ipynb

```diff
@@ -257,16 +275,42 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Submit the pipeline for execution"
+    "### Compile pipeline and submit for execution"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 19,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "Run link <a href=\"/pipeline/#/runs/details/7fe15fa4-df4f-11e9-8bf7-42010a8001b5\" target=\"_blank\" >here</a>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
    "source": [
-    "kfp.Client().create_run_from_pipeline_func(pipeline, arguments={}, experiment_name=EXPERIMENT_NAME)"
+    "# Specify pipeline argument values\n",
+    "arguments = {}\n",
+    "\n",
+    "# Submit a pipeline run\n",
+    "run_name = pipeline_func.__name__ + ' run'\n",
```

Comment:
Is there any reason to set the run name?

Reply:
Actually no. Removed.